rules/windows/defense_evasion_disabling_windows_defender_powershell.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2021/07/07"
+maturity = "production"
+updated_date = "2021/07/07"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies use of Set-MpPreference powershell command to disable or weaken certain Windows Defender preferences."
```

Comment:
Instead of preferences, could be say "security settings"?

Reply:
changed.